src/compressor/QatAccel.cc

```diff
@@ -48,22 +53,111 @@ bool QatAccel::init(const std::string &alg) {
   rc = qzSetDefaults(&params);
   if (rc != QZ_OK)
       return false;
+  return true;
+}
 
+static bool SetupSession(QatAccel::session_ptr &session, QzSessionParams_T &params) {
```

Comment:
please be consistent when naming functions. see https://github.com/ceph/ceph/blob/master/CodingStyle for more details.

Reply:
I have changed the name format. Thanks for reminding CodingStyle.